tests/test_symlinks.py

```diff
@@ -312,10 +311,10 @@ def test_update_file_to_symlink(tmp_path_factory: pytest.TempPathFactory) -> Non
 
     with local.cwd(src):
         # test updating a symlink
-        os.remove("bbbb.txt")
-        os.symlink("aaaa.txt", "bbbb.txt")
-        os.remove("cccc.txt")
-        with open("cccc.txt", "w+") as f:
+        os.remove("bbbb.txt")  # noqa: PTH107
+        os.symlink("aaaa.txt", "bbbb.txt")  # noqa: PTH107
+        os.remove("cccc.txt")  # noqa: PTH107
+        with open("cccc.txt", "w+") as f:  # noqa: PTH123
```

Comment:
Why are these not updated and ignored?

Reply:
i think i thought pathlib wasn't adding a lot of value here. it's not, but i think it's worth it for consistency. will update